build/DependencyVersions.props

```diff
@@ -13,7 +13,7 @@
     <NuGetVersion>4.3.0-preview2-4095</NuGetVersion>
     <NewtonsoftJsonVersion>9.0.1</NewtonsoftJsonVersion>
     <SystemReflectionMetadataVersion>1.4.2</SystemReflectionMetadataVersion>
-    <NETStandardLibraryNETFrameworkVersion>2.0.0-preview2-25331-02</NETStandardLibraryNETFrameworkVersion>
+    <NETStandardLibraryNETFrameworkVersion>2.0.0-preview2-25405-01</NETStandardLibraryNETFrameworkVersion>
```

Comment:
I just took the latest build out of `release/2.0.0`, I can choose a different version if one is more appropriate.  Let me ask around.